packages/components/Button/src/ToggleButton/ToggleButton.tsx

```diff
@@ -18,6 +19,7 @@ export const ToggleButton = compose<ToggleButtonType>({
     root: View,
     icon: Icon,
     content: Text,
+    shadow: Shadow,
```

Comment:
Added slot, but don't render it below? But also we might want to exclude this until we have a more general solution to not render a shadow if no token is specified?

Reply:
Yeah the way I set it up, things wouldn't build unless I added the slot everywhere...I can take another look to see if this can be optional